includes/multi-currency/CountryFlags.php

```diff
@@ -288,12 +289,12 @@ public static function get_by_country( string $country ): string {
 	 */
 	public static function get_by_currency( string $currency ): string {
 		$exceptions = [
-			'ANG' => '',
-			'BTC' => '',
-			'XAF' => '',
-			'XCD' => '',
-			'XOF' => '',
-			'XPF' => '',
+			Currency_Code::NETHERLANDS_ANTILLES       => '',
+			Currency_Code::BITCOIN                    => '',
```

Comment:
Do we need BTC here?

Reply:
It was there in the previous code so I assume it is BITCOIN xD

Reply:
@jessy-p @zmaglica what do you think about this? Should I remove it?

Reply:
I think we can remove BTC. For now, we don't plan to use it, and we can easily add it if it is needed.